src/copychars.js

```diff
@@ -9,10 +9,12 @@
  * @param {int} [length]
  * @return {string}
  */
-export default function copychars(string, start = 0, length) {
+export default function copychars(string, start = 0, length = undefined) {
   if (typeof string !== "string" || string === "") {
     return "";
   }
 
-  return (" " + string.substr(start, length)).substr(1);
+  const end = Number.isInteger(length) ? start + length : string.length;
+
+  return string.slice(start, end);
```

Comment:
But `slice` is what we're trying to fix with the `copyChar` isn't it? It uses `SlicedString` structure that goes out of hand in our situation.

Reply:
🤦‍♂️ ... forgot to stage a change before committing. By appending an empty character and then slicing the string as follows forces Chrome to "flat" copy the characters.
 ```js

(" " + string.slice(start, end)).slice(1);

```

Reply:
All good! This is what 2 x 👀 are for :)